apps/network-alerts/js/attention/attention.js

```diff
@@ -31,26 +30,24 @@ function sendNotification() {
   }
 
   return new Promise(function(resolve, reject) {
```

Comment:
After trying to work on that with the patch author, I believe we should not do this in this patch. The current code uses `reject` condition inside `notification.onerror`, which would be very hard to emulate without `new Promise` code.